src/components/Tabs/Tabs.tsx

```diff
@@ -161,21 +162,17 @@ export const Tabs = ({
       prevActiveIndex !== activeIndex
     ) {
       setActiveIndexState(activeIndex);
-      tabRefs[activeIndex].current.focus();
+      tabRefs[activeIndex].current?.focus();
     }
   }, [prevActiveIndex, activeIndex, tabRefs]);
 
   /**
    * Autogenerate ids on tabs if not defined.
    */
   useEffect(() => {
```

Comment:
Pretty sure none of these `useEffect`s in these files are working as expected. They're probably re-rendering unnecessarily (which may not really be that big of a deal). I'll look at that.